forc-util/src/lib.rs

```diff
@@ -78,7 +78,7 @@ pub fn validate_name(name: &str, use_case: &str) -> Result<()> {
             it conflicts with Forc's build directory names"
         );
     }
-    if name == "test" {
+    if name.to_lowercase() == "test" {
         bail!(
             "the name `test` cannot be used as a package name, \
```

Comment:
```suggestion
            "the name `test` cannot be used as a project name, \
```

Reply:
for sure! I'll update in a bit, thank you!